src/murmur/main.cpp

```diff
@@ -511,29 +511,29 @@ int main(int argc, char **argv) {
 
 	if (! Meta::mp.qsDBus.isEmpty()) {
```

Comment:
Probably not something for this issue but we should think about moving that initialization code into the DBus specific file. main is bloated enough as it is.